systemd/osdf-cache.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description = Pelican service osdf-cache
+After = network.target nss-lookup.target
+
+[Service]
+ExecStart = /usr/bin/pelican --config /etc/pelican/osdf-cache.yaml cache serve
```

Comment:
Let's use a binary named OSDF.  That turns on a bunch of default configs and other internal code paths.